build/docker/scripts/sca.sh

```diff
@@ -49,6 +49,10 @@ flake8 --ignore=E501 test/test.py
 flake8 --ignore=E501,E722 test/crossrunner
 flake8 test/features
 
+# PHP code style
+composer install --quiet
+./vendor/bin/phpcs
```

Comment:
@jeking3 
`phpcs` will check which PHP library match PSR-2.(like `flake8`)

Reply:
@RobberPhex this change is causing errors in other builds even though I have rebased on master, see:

https://travis-ci.org/apache/thrift/jobs/341109387

I am going to disable it for now.